src/web/www/style/variables.css

```diff
@@ -198,8 +204,9 @@
   /* BELOW CODE LEVEL */
   --err-hole-arms-z: 1;
   --tile-z: 2;
-  --err-hole-z: 3;
-  --select-z: 4;
+  --err-hole-z: 4;
+  --warn-hole-z: 3;
+  --select-z: 5;
```

Comment:
why this change? ie does anything go wrong if errors and warnings have the same z-index?